### PR TITLE
add doc about using kolla[-ansible]/master

### PR DESCRIPTION
**Pull request by msimonin**

### CODE REVIEW

File: docs/customization/index.rst [hunk 1]
```diff
@@ -39,7 +39,7 @@ Note on the network interfaces:
 
 Providers do their best to configure the network decently. This probably doesn't
 cover all the possible use cases. But, if you know what interfaces are configured by
-the provider you can specify a more precise allocation under he ``kolla`` key.
+the provider you can specify a more precise allocation under the ``kolla`` key.
 For instance :
```
Comment (rcherrueau) [inline docs/customization/index.rst]: Could you remove the extra space before `:'

File: docs/customization/index.rst [hunk 2]
```diff
@@ -49,6 +49,22 @@ For instance :
       neutron_external_interface: eth2
       tunnel_interface: eth3
 
+Running from kolla/master
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This can bite but you can run the latest Kolla code with the latest built
```
Comment (rcherrueau) [inline docs/customization/index.rst]: I would write, "if you want to live on the bleeding edge, you can run ..." but it's a personal opinion.

File: docs/customization/index.rst (hunk 1 above)
Comment (msimonin) [inline docs/customization/index.rst]: Done.

File: docs/customization/index.rst (hunk 2 above)
Comment (msimonin) [inline docs/customization/index.rst]: Done.

**Pull request merged**